From 3d99559779d628704568879a2ee51e968e66d005 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Tue, 5 Oct 2021 00:11:26 +0800
Subject: [PATCH] llvm_deps.cmake: don't copy header file when building

We build in pre-built mode and this header shouldn't be copied in
that case.

Upstream-Status: Pending

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 external/llvm/llvm_deps.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/external/llvm/llvm_deps.cmake b/external/llvm/llvm_deps.cmake
index 425d3766f..e43804f77 100644
--- a/external/llvm/llvm_deps.cmake
+++ b/external/llvm/llvm_deps.cmake
@@ -46,9 +46,9 @@ if(IGC_OPTION__LLVM_LLD)
   include(llvm_lld_source_hook)
   if(NOT EXISTS "${IGC_LLVM_WORKSPACE_SRC}/libunwind/include/mach-o" AND ${IGC_OPTION__LLVM_PREFERRED_VERSION} GREATER_EQUAL "12.0.0")
     # Need to copy one header from unwind package for LLD (only for building from sources)
-    file(MAKE_DIRECTORY ${IGC_LLVM_WORKSPACE_SRC}/libunwind/include/mach-o)
-    file(COPY ${DEFAULT_IGC_LLVM_SOURCES_DIR}/libunwind/include/mach-o/compact_unwind_encoding.h
-         DESTINATION ${IGC_LLVM_WORKSPACE_SRC}/libunwind/include/mach-o/)
+    #file(MAKE_DIRECTORY ${IGC_LLVM_WORKSPACE_SRC}/libunwind/include/mach-o)
+    #file(COPY ${DEFAULT_IGC_LLVM_SOURCES_DIR}/libunwind/include/mach-o/compact_unwind_encoding.h
+    #     DESTINATION ${IGC_LLVM_WORKSPACE_SRC}/libunwind/include/mach-o/)
   endif()
 endif()
 
-- 
2.32.0

